docs/interactions/Slash_Commands.md

```diff
@@ -0,0 +1,820 @@
+
+Edit a global commands. Updates will be available in all guilds after 1 hour. Returns `200` and an [ApplicationCommand](#DOCS_INTERACTIONS_SLASH_COMMANDS/application-command) object.
+
+###### JSON Params
+
+| Field       | Type                                                                                              | Description                    |
+|-------------|---------------------------------------------------------------------------------------------------|--------------------------------|
+| name        | string                                                                                            | 3-32 character command name    |
+| description | string                                                                                            | 1-100 character description    |
+| options?    | array of [ApplicationCommandOption](#DOCS_INTERACTIONS_SLASH_COMMANDS/application-command-option) | the parameters for the command |
+
+
+## Delete Global Application Command % DELETE /applications/{application.id}/commands/{id}
+
+Deletes a global commands. Returns `204`.
+
+## Get Guild Application Commands % GET /applications/{application.id}/guilds/{guild.id#DOCS_RESOURCES_GUILD/guild-object}/commands
+
+Fetch all of the global commands for your application. Returns a list of [ApplicationCommand](#DOCS_INTERACTIONS_SLASH_COMMANDS/application-command) objects.
+
+## Create Guild Application Command % POST /applications/{application.id}/guilds/{guild.id#DOCS_RESOURCES_GUILD/guild-object}/commands
+
+> danger
+> Creating a command with the same name as an existing command for your application will overwrite the old command.
+
+Create a new global commands. New global commands will be available in all guilds after 1 hour. Returns `200` and an [ApplicationCommand](#DOCS_INTERACTIONS_SLASH_COMMANDS/application-command) object.
+
+###### JSON Params
+
+| Field       | Type                                                                                              | Description                    |
+|-------------|---------------------------------------------------------------------------------------------------|--------------------------------|
+| name        | string                                                                                            | 3-32 character command name    |
+| description | string                                                                                            | 1-100 character description    |
+| options?    | array of [ApplicationCommandOption](#DOCS_INTERACTIONS_SLASH_COMMANDS/application-command-option) | the parameters for the command |
+
+## Edit Guild Application Command % PATCH /applications/{application.id}/guilds/{guild.id#DOCS_RESOURCES_GUILD/guild-object}/commands
+
+Edit a global commands. Updates will be available in all guilds after 1 hour. Returns `200` and an [ApplicationCommand](#DOCS_INTERACTIONS_SLASH_COMMANDS/application-command) object.
```

Comment:
same here, this shouldnt say "global commands", and the updates shouldnt take 1 hour either.